packages/core/admin/server/services/user.js

```diff
@@ -125,7 +126,7 @@ const resetPasswordByEmail = async (email, password) => {
  * @param {int|string} userId user's id to look for
  */
 const isLastSuperAdminUser = async userId => {
-  const user = await findOne({ id: userId }, ['roles']);
+  const user = await findOne(userId, ['roles']);
```

Comment:
if the defaultpopulate is roles you could remove it from the params

Reply:
I saw that too, let's do it